compose/local/app/start

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+
+python manage.py migrate
+python manage.py runserver_plus 0.0.0.0:8000
```

Comment:
how one would run the app in production?

Reply:
in production I would of course use either uwsgi or gunicorn HTTP Server